docs/DEVELOPMENT-GUIDE.md

```diff
@@ -226,6 +226,24 @@ Unit tests are run automatically with `make`. To run a subset of the unit tests,
 UNIT_TESTS=s2n_hash_test make
 ```
 
+### Debugging With GDB
+When trying to debug a failing test case, it is often useful to use a debugger like `gdb`. To do this, first make sure that the tests and s2n are compiled with debug information. This can be done by setting the CMAKE_BUILD_TYPE to DEBUG. Alternately, you can set the build type to R.
```

Comment:
```suggestion
When trying to debug a failing test case, it is often useful to use a debugger like `gdb`. To do this, first make sure that the tests and s2n are compiled with debug information. This can be done by setting the CMAKE_BUILD_TYPE to DEBUG. Alternatively, you can set the build type to R.
```

I'm not a cmake expert, but wouldn't setting the build type to R imply "Release" instead of "Debug"?

Reply:
Hmm, I now remember that I meant to look up the spelling of "RelWithDebInfo" or whatever the setting is, and apparently forgot to do that :(